packages/protocol/contracts/layer1/verifiers/compose/SgxAndZkVerifier.sol

```diff
@@ -22,20 +22,20 @@ contract SgxAndZkVerifier is ComposeVerifier {
         )
     { }
 
-    function areVerifiersSufficient(address[] memory _verifiers)
+    function areVerifiersSufficient(uint8[] memory _verifierIds)
         internal
         view
         override
         returns (bool)
     {
-        if (_verifiers.length != 2) return false;
+        if (_verifierIds.length != 2) return false;
 
-        if (_verifiers[0] == sgxRethVerifier) {
-            return _verifiers[1] == risc0RethVerifier || _verifiers[1] == sp1RethVerifier;
+        if (_verifierIds[0] == SGX_RETH) {
```

Comment:
since SGX_RETH < RISC0_RETH and SGX_RETH < SP1_RETH, the third if-statement will never be true -- as the input _verifierIds are in incremental order.